src/Elastic.CommonSchema.LoggerProvider/ConnectionPoolType.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to Elasticsearch B.V under one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+namespace Elastic.CommonSchema
```

Comment:
The namespace for all files should be `Elastic.CommonSchema.LoggerProvider`

Reply:
I wonder if `Elastic.CommonSchema.Extensions.Logging` would be a more descriptive name for this project. 


@russcam @gregkalapos thoughts?

Reply:
I actually want `Elasticearch` in the name to signal it does live ingestion of logs to `Elasticsearch` but `Elastic.CommonSchema.Extension.Logging.Elasticsearch` reads horribly. Anyone willing to bikeshed this one ?

Reply:
Naming is important.

Probably don't want to repeat the class suffix in the namespace (also consider the 'using ...' statement needed for the LoggingBuilder extensions): Elastic.CommonSchema.LoggerProvider.ElasticsearchLoggerProvider 

I used Elastic.CommonSchema to be consistent with the other project components, and didn't want it too long, but the project/package name ended up inconsistent with the namespace.

If you want other suggestions, when searching for the library, people may use terms like "Elasticsearch LoggerProvider", and shorter is better:
* using Elastic.CommonSchema.Logging; & class Elastic.CommonSchema.Logging.ElasticsearchLoggerProvider
* using Elastic.Logging; & class Elastic.Logging.ElasticsearchLoggerProvider
* using Elastic.Extensions.Logging; & class Elastic.Extensions.Logging.ElasticsearchLoggerProvider

.NET uses Microsoft.Extensions.Logging.Console.ConsoleLoggerProvider, but the ConsoleLoggerExtensions only needs "using Microsoft.Extensions.Logging;", hence my initial use of just "using Elastic.CommonSchema:" at least for the builder.

Reply:
Happy to change this if you decide. Elastic.Extensions.Logging?